src/init.cpp

```diff
@@ -1016,14 +1016,33 @@ bool AppInitParameterInteraction()
         CAmount n = 0;
         if (!ParseMoney(GetArg("-minrelaytxfee", ""), n) || 0 == n)
             return InitError(AmountErrMsg("minrelaytxfee", GetArg("-minrelaytxfee", "")));
-        // High fee check is done afterward in CWallet::ParameterInteraction()
         ::minRelayTxFeeRate = CFeeRate(n);
     } else if (incrementalRelayFee > ::minRelayTxFeeRate) {
         // Allow only setting incrementalRelayFee to control both
         ::minRelayTxFeeRate = incrementalRelayFee;
         LogPrintf("Increasing minrelaytxfee to %s to match incrementalrelayfee\n",::minRelayTxFeeRate.ToString());
     }
 
+    // This is the maximum absolute fee (in COIN, not satoshis)
```

Comment:
Tiny nit: would this comment be better with "koinu" instead of "satoshis"?

Reply:
For now I'd not make this blocking because, for example:

```console
$ grep -rn satoshi src/validation*
src/validation.h:62://! Discourage users to set fees higher than this amount (in satoshis) per kB
src/validation.h:68://! -maxtxfee will warn if called with a higher fee than this amount (in satoshis)
src/validation.h:187:/** Absolute maximum transaction fee (in satoshis) used by wallet and mempool (rejects high fee in sendrawtransaction) */
```

Let's fix that for 1.21 the moment we start with "branding"?

Reply:
That plan makes sense to me.